packages/dev/package.json

```diff
@@ -15,8 +15,10 @@
     "truffle-compile": "truffle compile",
     "lint": "eslint -f unix .",
     "test": "lerna run tsc && truffle --compile-all --network npmtest test",
+    "truffle-console": "truffle console",
     "truffle-test": "truffle test --compile-all ",
     "test1": "truffle --network npmtest test",
+    "test2": "hardhat --network npmtest test",
```

Comment:
this is getting out of control...

Reply:
That I can remove for now, I tried running hardhat as test framework instead of truffle as well, and that's a left over from that.

Reply:
please do, lets at least try to keep track of adding new yarn tasks))